src/ui/localization/ui-translator.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import { clone, extend, warnOnce } from '../../util/util';
+import uk from './uk';
```

Comment:
As there aren't a lot of strings to deal with (yet), what about putting all the localizations in a single module, as [mapbox-gl-geocoder has done](https://github.com/mapbox/mapbox-gl-geocoder/pull/201/files#diff-76473b4e518211fdbb427d6d2e0127d3)?

Importing that one module would then give us the equivalent of the `SupportedUiLanguages` object here, and in the future we wouldn't have to touch this file to add new imports every time we add a language.